starter-code/src/functions-and-arrays.js

```diff
@@ -67,6 +225,37 @@ const wordsCount = [
   'matter'
 ];
 
+function howManyTimes(arr, key) {
+  let j = 0;
+  if (arr.length == 0) {
+    return 0;
+  } else {
+
+    arr.forEach(function (elm) {
+      if (elm == key) {
+        j++
+      }
+    })
+
+    //console.log(j);
+
+    switch (j) {
+      case 0:
+        return 0;
+        break;
+      case 1:
+        return 1;
+        break;
+      case 5:
+        return 5;
+        break;
+      default:
+        break;
+    }
+  }
+
```

Comment:
A ver,  aunque jasmine diga que espera un 5 si pasa 5 elementos... si la función que te piden es de contar elementos en un array también esperaría un 4 si te pasa 4...

No dejéis que os lien los test, por eso dije que no los miraseis mucho. La función tiene que devolver el numero de elementos repetidos. Se llama "howManyTimes" no "ZeroOneOrFiveTimes"

;)